Add Project Loved: August 2021 news post (#5978)
ppy/osu-wiki: news/2021-08-19-project-loved-august-2021.md

[![](/wiki/shared/news/2021-08-19-project-loved-august-2021/534353.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})

[kors k - Shibuya Jungle (lapix Remix)](https://osu.ppy.sh/beatmapsets/534353#osu) by [Akali](https://osu.ppy.sh/users/2127359)\
146 BPM, 6:41 | 8.21★\
*description written by [Bubbleman](https://osu.ppy.sh/users/5182050)*

If you've hung around in a top player's stream before, chances are you'll have seen this map at one stage or another. Submitted in late 2016, this is one of Akali's most challenging maps still to this date. The bulk of the map contains 146 BPM large spacing technical patterns that sometimes turn into jumps and are interspersed with bursts at 292 BPM. Hence, the mechanical difficulty of this map is definitely not to be messed with, especially when it doesn't give you any downtime over its almost 7-minute length. Think you're up to the challenge?

## osu!taiko

> osu!taiko Loved candidates were chosen by [-Kazu-](https://osu.ppy.sh/users/920861) and [X a v y](https://osu.ppy.sh/users/3738344)!

Reviewer: ```suggestion
The osu!taiko Loved candidates were chosen by [-Kazu-](https://osu.ppy.sh/users/920861) and [X a v y](https://osu.ppy.sh/users/3738344)!
```

*description written by [X a v y](https://osu.ppy.sh/users/3738344)*

At 240 BPM, *kedamono_friends* is quite an upbeat and enjoyable piece of work, featuring a variety of comfortable patterns tossed in with a lot of short 1/6 bursts. The map maintains the same intensity for 4 minutes, which makes it quite difficult yet helpful for advanced players who want to practice hitting short bursts.

[![](/wiki/shared/news/2021-08-19-project-loved-august-2021/1530135.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})

[Mono. - Meteor Xpress](https://osu.ppy.sh/beatmapsets/1530135#taiko) by [Shamirin_mzk](https://osu.ppy.sh/users/11325757)\
190 BPM, 2:31 | 6.64★\
*description written by [X a v y](https://osu.ppy.sh/users/3738344)*

This is the latest edition to Shamirin_mzk's "Mono." series, which was received surprisingly well from the playerbase. In the series, Shamirin_mzk uses his special craft to make creative barline gimmicks that fit very well with the artist's music. Moreover, this map uses a gimmick that has not been seen before in this series and makes sure that every single moment feels engaging and never fails to show off its aesthetics.

## osu!catch

> osu!catch Loved candidates were chosen by [Sartan](https://osu.ppy.sh/users/4100941), [Spectator](https://osu.ppy.sh/users/702598), [Tenshichan](https://osu.ppy.sh/users/1101600) and [Wesley](https://osu.ppy.sh/users/2407265)!

Reviewer: ```suggestion
The osu!catch Loved candidates were chosen by [Sartan](https://osu.ppy.sh/users/4100941), [Spectator](https://osu.ppy.sh/users/702598), [Tenshichan](https://osu.ppy.sh/users/1101600) and [Wesley](https://osu.ppy.sh/users/2407265)!
```

The original AR of the map is 9.3 but the set also comes with an AR 9.6 difficulty for people who are more comfortable with a higher AR. Make sure to check it out!

[![](/wiki/shared/news/2021-08-19-project-loved-august-2021/186074.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})

[yuikonnu - Souzou Forest](https://osu.ppy.sh/beatmapsets/186074#fruits) by [Serena-](https://osu.ppy.sh/users/2193444) and [Kowaii_ne](https://osu.ppy.sh/users/2213794)\
180 BPM, 3:38 | 1.98★, 3.16★, 4.61★\
*description written by [Sartan](https://osu.ppy.sh/users/4100941)*

Sometimes, and I mean *sometimes*, you want something that is calm, not too straining, chill, you name it. Well, here you go! In this unusual not so 'pro' level collab of two ex-pro players, one can really take time to enjoy themselves for a while. Having only a Salad, Platter, and Rain difficulty makes this map playable for any player; whether you have thousands of hours of experience or only one. Soooo... Take it easy, vibe a little to *Souzou Forest*, and have a good one!

## osu!mania

> osu!mania Loved candidates were chosen by [\_underjoy](https://osu.ppy.sh/users/2235750), [-mint-](https://osu.ppy.sh/users/8976576), [Kawawa](https://osu.ppy.sh/users/4647754) and [Penguinosity](https://osu.ppy.sh/users/10235296)!

Reviewer: ```suggestion
The osu!mania Loved candidates were chosen by [\_underjoy](https://osu.ppy.sh/users/2235750), [-mint-](https://osu.ppy.sh/users/8976576), [Kawawa](https://osu.ppy.sh/users/4647754) and [Penguinosity](https://osu.ppy.sh/users/10235296)!
```

The people who provided beatmaps to this set are well-known within the 6K mapping scene and the mapping community in general, and we're thankful that Imperial Wolf put this together as it's a pretty cohesive rice-biased pack with a good difficulty spread. Make sure to check out all of the mappers who've helped put this together, and make sure to send your vote!

[![](/wiki/shared/news/2021-08-19-project-loved-august-2021/655302.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})

[YUC'e - Future Cider](https://osu.ppy.sh/beatmapsets/655302#mania) by [bsc](https://osu.ppy.sh/users/6928171), [Arzenvald](https://osu.ppy.sh/users/3027421), [Bpple](https://osu.ppy.sh/users/8210168) and [Jakads](https://osu.ppy.sh/users/259972)\
180 BPM, 3:59 | 1.06★ – 7.37★\
*description written by [\_underjoy](https://osu.ppy.sh/users/2235750)*

> *Future Cider* is a mapset that has been there for quite a long time, and it has even been showcased in the Finals Mappool of the [osu!mania 7K World Cup 2018](/wiki/Tournaments/MWC/2018_7K). Since then it has certainly become a crowd favourite, gaining significant traction.

Reviewer: you either add "tournament" after or remove the "the" (or hypothetically remove "2018")

```suggestion
*Future Cider* is a mapset that has been there for quite a long time, and it has even been showcased in the Finals Mappool of [osu!mania 7K World Cup 2018](/wiki/Tournaments/MWC/2018_7K). Since then it has certainly become a crowd favourite, gaining significant traction.
```

Author: i'll remove "the". will fix in forum post commit.

This map is already near 6 years old but still a hilarious demonstration of just how far you can take the O2Jam-styled patterning for a song like this. The intro to the chart is pretty tame, but once it's over, you're quickly thrown into what I can only describe as "LN Art". Full LN double stairs moving from left to right, uncomfortable anchored LN holds and gaps, and on top of that, almost all of this is charted to 1/8. For the map being just over a minute thirty, Cuppp managed to pack in more pain and suffering than you'll see in most maps of this vein more than triple its length. Now onto gemboyong's 4K difficulty, they used Cuppp's map as the blueprint and manually converted any of the patterns that could comfortably fit with the lesser note count to 4K. Many 4K players should be aware of this diff specifically since it's featured in their popular [Gems pack 13](https://osu.ppy.sh/beatmapsets/555819) that was uploaded in 2017.

> Anyways, I'm glad we were able to get gemboyong's difficulty included, and the Loved team is happy to be finally be bringing you these masterpieces. LET'S GO!

Reviewer: ```suggestion
Anyways, I'm glad we were able to get gemboyong's difficulty included, and the Loved team is happy to finally bring you these masterpieces. LET'S GO!
```

> Anyways, I'm glad we were able to get gemboyong's difficulty included, and the Loved team is happy to be finally bring you these masterpieces. LET'S GO!

Reviewer: always copy-paste the whole line if you have to manually apply changes

```suggestion
Anyways, I'm glad we were able to get gemboyong's difficulty included, and the Loved team is happy to finally bring you these masterpieces. LET'S GO!
```

Author: whoops, will be fixed when given the all-clear for forum posts

[![](/wiki/shared/news/2021-08-19-project-loved-august-2021/1039602.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})

[GIRAFFES? GIRAFFES! -  I Am S/H(im)e\[r\] As You Am S/H(im)e\[r\]](https://osu.ppy.sh/beatmapsets/1039602#mania) by [\_underjoy](https://osu.ppy.sh/users/2235750)\
264 BPM, 9:26 | 4.95★\
The \[Davvy's Dreaming Descent\] difficulty is *not* being nominated for Loved.\
*description written by [\_underjoy](https://osu.ppy.sh/users/2235750)*

> \_underjoy's 4K difficulty of *I Am S/H(im)e\[r\]* is a solid technical hybrid map with difficulty predominantly located in the beginning and ending of the song. The marathon starts with a high speed repeated jumpstream section, which is repeated throughout the chart. Then, long note parts with tricky releases and some more some irregular rhythms are introduced that pose a significant accuracy challenge. The intensity is abruptly stopped and player can rest while fully immersing themselves in the calm nature of the song. Long notes and chords are patterned in a very recurrent manner that emphasizes constant repetition and overall theme of the chart. A long break introduces the player to the last part, which is a slightly modified version of the first one.

Reviewer: ```suggestion
\_underjoy's 4K difficulty of *I Am S/H(im)e\[r\]* is a solid technical hybrid map with difficulty predominantly located in the beginning and ending of the song. The marathon starts with a high-speed jumpstream section, which is repeated throughout the chart. Then, long note parts with tricky releases and some more some irregular rhythms are introduced, posing a significant accuracy challenge. The intensity is abruptly stopped and the player can rest while fully immersing themselves in the calm nature of the song. Long notes and chords are patterned in a very recurrent manner that emphasizes constant repetition and overall theme of the chart. A long break introduces the player to the last part, which is a slightly modified version of the first one.
```
